### ISSUE TITLE
returns false for a valid ipfs url/path!

### ISSUE THREAD

**Issue by xylanreeves** [ipfs-shipyard/is-ipfs#48]
isIPFS.urlOrPath("ipfs://bafybeihbsysdkemc3kyylegtfopkrcfiih4exnasoql2q36fb4zawlrwhy/volcano.json")

returns false when it is a valid ipfs url/path!

**Comment by TimDaub** [comment]
yeah I was wondering the same thing. I'm currently crawling Zora's NFT contracts and many image URIs are the IANA ipfs scheme `ipfs://` and I can't find a good way of "just" downloading them by looking at the docs. Am I missing something?

**Comment by NCGThompson** [comment]
I don't remember where, but I did read somewhere that `ipfs:`(without the `//`) and `ipfs:/` should also be interpreted as ipfs urls. The same goes for `ipns://`.

In addition, `dweb://ipfs/` and `dweb://ipns/` (or something like them) are also ipfs urls.
